set.mm

```diff
@@ -43972,6 +43985,24 @@ practical reasons (to avoid having to prove sethood of ` A ` in every use
       DTUNULCEABFDPRUIUOBFUNBCEBEUAABFUBUCBFUDAUNCEABFUERUFUGUH $.
   $}
 
+  ${
+    $d w x y A $.  $d x y z A $.  $d y z ph $.  $d w ph $.  $d z ps $.
+    $d w ps $.  $d y ch $.
+    reu8nf.1 $e |- F/ x ps $.
+    reu8nf.2 $e |- F/ x ch $.
+    reu8nf.3 $e |- ( x = w -> ( ph <-> ch ) ) $.
+    reu8nf.4 $e |- ( w = y -> ( ch <-> ps ) ) $.
+    $( Restricted uniqueness using implicit substitution.  This version of
+       ~ reu8 uses a bound-variable hypothesis for ` x ` and ` ps ` instead of
```

Comment:
I suggest: ...uses a non-freeness hypothesis for...
Since we generally refer to F/ as the non-freeness predicate, this might make searches easier (and there is always the fact that our definition is not syntactic e.g. x is non-free in our sense in x=x, although it is not bound either).

Reply:
I copied the text from another comment, but I agree that your suggestion is better. I'll change it here, but it should also be changed in other comments (later).

Reply:
Done